src/mailing/sender.js

```diff
@@ -31,4 +34,34 @@ async function sendTestEmail(recipient) {
   });
 }
 
-module.exports = sendTestEmail;
+async function sendForgotPasswordEmail(toEmail, resetToken) {
+  const transporter = nodemailer.createTransport({
+    service: 'gmail',
+    auth: {
+      user: process.env.EMAIL_USER,
+      pass: process.env.EMAIL_PASSWORD,
+    },
+  });
+
+  // Load and render the HTML template
+  const htmlTemplate = await ejs.renderFile(
+    path.join(__dirname, './email/templates/html.ejs'),
```

Comment:
```suggestion
    path.join(__dirname, './templates/forgot_password/html.ejs'),
```

![](https://github.com/Code-the-Dream-School/dd-prac-team4-back/assets/9094098/11f9f7a0-f42d-4a90-84a1-1ed3aba402ef)